lexxpluss_apps/src/imu_controller.cpp

```diff
@@ -23,92 +23,202 @@
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include <zephyr.h>
+#include <drivers/gpio.h>
 #include <device.h>
 #include <drivers/sensor.h>
 #include <logging/log.h>
 #include <shell/shell.h>
 #include "imu_controller.hpp"
 #include "runaway_detector.hpp"
 
+#define PI 3.14159265358979323846
```

Comment:
math.hで定義済みのM_PIを使用するほうがポータビリティに優れると思いました。
I thought it would be more portable to use M_PI, which is already defined in math.h.

https://docs.zephyrproject.org/apidoc/latest/math_8h.html

Reply:
math.hを使おうとしましたが、runaway_detectorがcommon.hppのM_PIを使っていたので、ひとまずそれに合わせました。